package.json

```diff
@@ -47,7 +47,7 @@
     "percy": "percy exec -- node snapshots.js",
     "icon-svgs-to-mixins": "node scripts/convert-svgs-to-icon-mixins.js icons"
   },
-  "version": "3.9.0",
+  "version": "3.9.1",
```

Comment:
We haven't released 3.9 yet, so that won't be necessary. 

(sorry, I know keeping up with that is confusing)